src/app/components/charts/top-compounds-list/top-compounds-list.component.ts

```diff
@@ -0,0 +1,161 @@
+import { Component, ElementRef, ViewChild,  AfterViewInit,
+         Input, ViewEncapsulation, ChangeDetectorRef,
+         ViewContainerRef } from '@angular/core';
+
+import { MdDialog, MdDialogConfig, MdDialogRef } from '@angular/material';
+import { Settings, AnnotatedPlatewellid, Zhang } from '../../../models';
+import { ActionDialogComponent } from './action-dialog/action-dialog.component';
+
+import * as d3 from 'd3';
+import 'd3-color';
+import 'd3-scale';
+
+const detailDialog = ActionDialogComponent;
+
+/* app colors definition */
+const appColors = [
```

Comment:
create a global static class for these values, they're repeated in different places.